src/web/routes.rs

```diff
@@ -181,7 +181,15 @@ pub(super) fn build_axum_routes() -> AxumRouter {
             get_internal(super::crate_details::crate_details_handler),
         )
         .route(
-            "/:name/releases",
+            "/platforms/:name/:version/:target/",
```

Comment:
Good point!

Reply:
Using `/-/` as prefix was problematic for tests so instead I used `/menus/` which actually renders pretty well I think.

Reply:
That would similarly cause an issue for <https://docs.rs/menus> if someone ever publishes it. What about `/-/` caused issues for tests? It shouldn't be treated specially in any way.

Reply:
It failed to query URLs starting with `/-/` saying they're blacklisted. Since normally it's used for static resources, I thought it was voluntary. Should I disable that?

Reply:
It's for any internal paths so that we don't have more routes that overlap with rustdoc routes, the only place it's blacklisted is in the rustdoc handler, I did a quick test changing it and it seems to work fine.

Reply:
Perfect, going to revert the URLs to use `/-/` and update the rustdoc handler.